Cargo.toml

```diff
@@ -846,7 +846,7 @@ macro_magic = { version = "0.5.1" }
 maplit = { version = "1.0.2" }
 memmap2 = { version = "0.9.3" }
 memory-db = { version = "0.32.0", default-features = false }
-merkleized-metadata = { version = "0.1.0" }
+merkleized-metadata = { git = "https://github.com/lexnv/merkleized-metadata.git", branch = "lexnv/update-frame-md" }
```

Comment:
```suggestion
merkleized-metadata = "0.1.1"
```

Reply:
We'll need another release for it once we merge: https://github.com/paritytech/frame-metadata/pull/86, sorry for the back and forth, wanted to add `transaction_extensions_by_version` as well 🙏